.gitignore

```diff
@@ -0,0 +1,8 @@
+dbt_packages/
+logs/
+env/
+target/
+.python-version
+venv/
+*.egg-info
+.user.yml
```

Comment:
I would also throw in:
```
dbt_modules/
dbt_packages/
*.DS_Store
```

First two are related to dbt Cloud: each dev must install packages/modules to their dbt Cloud IDE individually, and these packages/modules will be stored in the `dbt_packages` and `/dbt_modules` directories, we can't just sync them in the repo itself.

Last one because Macs are dumb and generate`DS_Store` files 😆